### ISSUE TITLE
Validation accuracy is not consistent

### ISSUE THREAD

**Issue by Lawhy** [OpenNMT/OpenNMT-py#1509]
Hi, I have encountered a problem with the validation accuracy.
In the log below, the validation acc is 91.71, but after I translate the validation set using that particular checkpoint, the acc becomes around 78, I am not sure what happened exactly? Is there any special calculation involved in computing validation acc?
![2](https://user-images.githubusercontent.com/33579950/61550462-67f63000-aa4a-11e9-8222-be3614174190.PNG)

**Comment by vince62s** [comment]
this question has been answered already is some threads.
https://github.com/OpenNMT/OpenNMT-py/issues/1457
